tests/appsec/test_automated_login_events.py

```diff
@@ -0,0 +1,175 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022 Datadog, Inc.
+
+from utils import weblog, interfaces, context, missing_feature, released, scenarios
+
+
+@released(cpp="?", golang="?", java="?", nodejs="?", dotnet="?", php="?", ruby="?")
```

Comment:
It's actually not implemented in node ?

Reply:
It's implemented but no released. I've tested it locally.

Reply:
@cbeauchesne no we don't release features without passing the system-tests first xD